dev/com.ibm.ws.request.timing/src/com/ibm/ws/request/timing/probeExtensionImpl/SlowRequestProbeExtension.java

```diff
@@ -75,6 +78,16 @@ public void processExitEvent(Event event, RequestContext requestContext) {
 			//Clean up activity for this request will be performed in the corresponding slow request timer task.
 
 			/** Instead of this should we just return false from invokeForEventExit? */
+		} else {
+			if ( requestContext.getisSlow()  &&  tc.isWarningEnabled()  &&  TraceComponent.isAnyTracingEnabled()  &&  tc.isDebugEnabled() ) { //if requestContext is a slowRequestContext
```

Comment:
you're just checking if you can write an info message.  Normally we don't bother with checking if that level is enabled since it's enabled by default and since Tr.info will double check the level anyway... so this should just be `if (requestContext.isSlow())`